back/engines/commercial/analytics/app/services/analytics/matomo_data_importer.rb

```diff
@@ -118,7 +125,8 @@ def persist_visits_locales(visit_data, visit_ids)
     def update_referrer_types(visit_data)
       referrer_types_attrs = visit_data
         .pluck('referrerType', 'referrerTypeName').uniq
-        .select { |type_infos| type_infos.none?(&:blank?) } # being extra cautious and checking that matomo doesn't return empty values
+        # being extra cautious and checking that matomo doesn't return empty values
```

Comment:
Reverted this to keep the comment where it was previously as the stack trace seemed to be grumbling about this function - although I don't think this was the issue